### ISSUE TITLE
[Android] [0.28.0] [BUG] First Picker.Item initiate reload cycle with onValueChange

### ISSUE THREAD

**Issue by cablegunmaster** [facebook/react-native#8560]
I am using the picker from the 0.28.0 Library and this version keeps reloading the first pickerItem is picked. Now I have to 'protect' my 'getdata' from initiating until something else is picked. (workaround)

First value will be set everytime I reload this component. If picked it will keep ´reloading' . 

```
export default class extends Component {
    constructor(props, state){
        super(props, state);
        this.state = {
            resolution: 'month',
        };
    };

requestingData(resol){
        this.setState({resolution: resol});
        this.getData(); //do some retrieval of data.
    }

render() {
<Picker
                        selectedValue={this.state.resolution}
                        onValueChange={(resol) => this.requestingData(resol) } >
                        <Picker.Item label="-" value="-" />
                        <Picker.Item label="Maand" value="month" />
                        <Picker.Item label="Jaar" value="year" />
                        <Picker.Item label="week" value="week" />
                    </Picker>
}
}
```

TL DR; The first  Picker.item is causing reload loops xD. 

Temp workaround:

```
if(resol !== '-' ) { 
    this.getData(); //do some retrieval of data.
}
```

Why is this bug annoying?
It keeps reloading and reloading and reloading, and you have no idea whether its your code which is broken or a variable is not set somewhere. until I tinkered with Picker.

**Comment by wfel** [comment]
+1

**Comment by KGALLET** [comment]
+1

**Comment by suihalee** [comment]
+1

**Comment by CapitanRedBeard** [comment]
https://github.com/facebook/react-native/commit/a2a8d7f5da93ee10cf5eccfb944f581f43cfe0c0
So v0.33 is suppose to fix this, but the same problem exists for me! Also this problem only seems to occur on the initial render of the page.

**Comment by lacker** [comment]
This should be fixed.
